airbyte-integrations/connectors/destination-s3/finalize_build.sh

```diff
@@ -7,7 +7,7 @@ ARCH=$(uname -m)
 
 if [ "$ARCH" == "x86_64" ] || [ "$ARCH" = "amd64" ]; then
   echo "$ARCH"
-  yum install lzop lzo lzo-dev -y
+  yum install lzop lzo lzo-devel -y
```

Comment:
just curious, what's the difference?

Reply:
Oh, `lzo-dev` simply doesn't exist on the amazonlinux distro we have on the base image, so the connector couldn't be built for amd64 platforms at all. This change is unrelated to the rest of the PR, AFAIK it's been broken for a while.